query-algebrizer/src/cc.rs

```diff
@@ -1235,4 +1235,114 @@ mod testing {
         // The type of the provided binding doesn't match the type of the attribute.
         assert!(cc.is_known_empty);
     }
+
+    #[test]
+    fn test_fulltext_type_disagreement() {
```

Comment:
Let's start `/// Noting what the test verifies`.  I'm guilty of not doing this as well.